lua/astrocommunity/debugging/persistent-breakpoints-nvim/init.lua

```diff
@@ -1,5 +1,6 @@
 return {
   "Weissle/persistent-breakpoints.nvim",
+  event = "BufReadPost",
   opts = function(_, opts)
     return require("astronvim.utils").extend_tbl(opts, {
       load_breakpoints_event = { "BufReadPost" },
```

Comment:
I kept it the same as this. Unsure what is better. I'll leave up to your hands

Reply:
Let's stick with this. :)